### PR TITLE
8 correct terminology

### PR DESCRIPTION
**Pull request by cchristelis**
This PR fixes terminology issues. The actual "Capture Init" and "Capture Done" functions are still placeholders.

### CODE REVIEW

File: src/katsdpk8spoc/sdpcontroller.py [hunk 1]
```diff
@@ -360,8 +488,10 @@ def main():
     swagger.add_routes(
         [
             web.get("/", home_page),
-            web.get("/start", start_handle),
-            web.get("/stop", stop_handle),
+            web.post("/product-configure", product_configure_handle),
+            web.get("/capture-init", capture_init_handle),
+            web.get("/capture-done", capture_done_handle),
```
Comment (martinslabber) [inline src/katsdpk8spoc/sdpcontroller.py]: would POST action not make this more consistent. Lets make the capture-* and deconfigure POST then its the same as product-configure.

File: src/katsdpk8spoc/sdpcontroller.py [hunk 2]
```diff
@@ -70,28 +79,76 @@ async def _stop_workflow(self, workflow_name: str):
...
+    def calculate_batch_limit(self):
+        """This is the batch job size limit. For now we are returning 10,
+        but later on we might have a better number.
+        The exact number of batch jobs depends on the, for now, "random" number
+        obtained in the Calibrator and Ingest nodes."""
+        return 10
+
+    @staticmethod
+    async def argo_get(url, headers=None):
+        """Get an ARGO Json to an URL"""
```
Comment (martinslabber) [inline src/katsdpk8spoc/sdpcontroller.py]: s/Json/JSON/

File: src/katsdpk8spoc/sdpcontroller.py [hunk 3]
```diff
@@ -116,16 +179,38 @@ async def status(self, subarray):
     async def check(self):
         pass
 
+    def get_antennas(self):
```
Comment (martinslabber) [inline src/katsdpk8spoc/sdpcontroller.py]: Let's just stick to receptors.  please rename to get_receptors

**Review by martinslabber — APPROVED** [review]
I still have holiday brain but this looks good.

File: src/katsdpk8spoc/sdpcontroller.py (hunk 1 above)
Comment (cchristelis) [reply src/katsdpk8spoc/sdpcontroller.py]: done

**Pull request merged**